backend/wmg/data/load_cube.py

```diff
@@ -1,19 +1,82 @@
-def update_s3_resources():
+import time
+import boto3
+import os
+import subprocess
+
+from backend.wmg.data.snapshot import CELL_TYPE_ORDERINGS_FILENAME
+
+
+stack_name = os.environ.get("REMOTE_DEV_PREFIX")
+wmg_bucket_name = os.environ.get("WMG_BUCKET")
+artifact_bucket_name = os.environ.get("ARTIFACT_BUCKET")
+
+
+def _get_wmg_bucket_path():
+    if stack_name:
+        return f"s3://{wmg_bucket_name}{stack_name}"
+    else:
+        return f"s3://{wmg_bucket_name}"
+
+
+def update_s3_resources(corpus_name):
     """
     Copy cube and cube related files to s3 under the current timestamp
     """
-    pass
+    timestamp = int(time.time())
```

Comment:
we could refactor to create the timestamp at the beginning of the pipeline run and then store all the created artifacts under the timestamp and then we would only need to run the sync command once to copy everything over

Reply:
We can do that easily, my only concern is whether that's also gonna copy temporary files created during the process.

Reply:
Just updated the pipeline code take in a path to make testing easier. If a path isnt given it will create one from the curr dir (which is what it currently uses) and a timestamp